Add example connection using web3j (#182)
ConsenSys/doc.goquorum: docs/concepts/multi-tenancy.md

1. HTTP header. Every RPC request must have an HTTP header "PSI" attached that specifies the private state to use.

1. IPC and inproc connections. Prepend the PSI to the ID field of the `jsonrpcMessage`.

### Accessing a private state using web3j

> To access private state using web3j you need to specify the query parameter in the URL when initialising the `HttpService`. It's also likely that you will have auth enabled and will need to specify a bearer token in the `Authorization` header.

Reviewer: ```suggestion
To access private state using [web3j](https://github.com/web3j/web3j), specify the query parameter in the URL when initializing the `HttpService`. If you have authentication enabled you must specify a bearer token in the `Authorization` header.
```

> To access private state using [web3j](https://github.com/web3j/web3j), specify the query parameter in the URL when initializing the `HttpService`. If you have authentication enabled you must specify a bearer token in the `Authorization` header.

Reviewer 2: ```suggestion
To access a private state using [web3j](https://github.com/web3j/web3j), specify the query parameter in the URL when initializing the `HttpService`. If you have authentication enabled you must specify a bearer token in the `Authorization` header.
```
